### PR TITLE
Add a call for Adopters

### LINKED ISSUE #366: Add a call for Adopters
**Current State of the Meshery.io site**
The Created By and Used By section of the site has a list of logos, but does not call for others to add their logo.

**Desired State of the Meshery.io site**
Add a call to action like "Add yourself as an adopter... and share your service mesh story with the community".

Screen Shot 2020-10-29 at 4 25 58 PM

Button should link to https://github.com/layer5io/meshery/blob/master/ADOPTERS.md

---
**Contributor Resources**
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md.

### PR DESCRIPTION
**Pull request by jlcastrogro**
Signed-off-by: José Luis Castro Guerrero <jlcastrogro@gmail.com>

**Description**

This PR fixes #366

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Neilblaze — APPROVED** [review]
LGTM :+1: 
Thank you @jlcastrogro :smiley:

**Comment by Neilblaze** [comment]
@leecalcote Also I think a little bit of styling needs to be updated in terms of mobile view. I am sharing a screenshot below.

meshery_mob

Here, we need to make certain changes.

- Increase a bit of whitespace under `USED BY` heading.
- Uplift Bookmyshow to align horizontally with Solarwinds.
- Place Ericsson at the center. [Optional since joining of a new adopter will fix this issue!]

File: _sass/landing-page.scss [hunk 1]
```diff
@@ -106,6 +106,35 @@
...
+  margin: auto;
+  padding: 0 0 20px 0;
+  text-align: center;
+
+  .adopter-title, .adopter-description {
+    color: dimgray;
+    font-weight: 600;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
```
Comment (JubayerJoy) [inline _sass/landing-page.scss:120]:
```suggestion
```

File: _sass/landing-page.scss [hunk 2]
```diff
@@ -106,6 +106,35 @@
...
+
+  .adopter-title {
+    margin-bottom: 0px;
+    font-size: 1.5em;
+  }
```
Comment (JubayerJoy) [inline _sass/landing-page.scss:125]:
```suggestion
  .adopter-title {
    font-size: 1.5em;
    font-weight: 500;
    margin: 0 0 10px 0;
  }
```

File: _sass/landing-page.scss [hunk 3]
```diff
@@ -106,6 +106,35 @@
...
+    margin-top: 0px;
+    margin-bottom: 30px;
+    font-size: 0.9em;
+  }
+
+  .adopter-button {
+    font-size: medium;
+  }
+}
+
```
Comment (JubayerJoy) [inline _sass/landing-page.scss:137]:
```suggestion

  .adopter-button {
    font-size: medium;
    margin-bottom: 50px;
  }
}

```

File: index.html [hunk 4]
```diff
@@ -196,6 +196,18 @@ <h1 class="used-by">Used By</h1>
 		</div>
 	</section>
 
+	<!-- Adopter Section -->
+	<section class="adopter-section">
+		<div class="adopter-container">
+            <h3 class="adopter-title">Add yourself as an adopter...</h3>
```
Comment (JubayerJoy) [inline index.html:202]:
```suggestion
            <h3 class="adopter-title">Add yourself as an <strong>adopter</strong></h3>
```

File: index.html [hunk 5]
```diff
@@ -196,6 +196,18 @@ <h1 class="used-by">Used By</h1>
...
+            <h5 class="adopter-description">and share your service mesh story with the community</h5>
```
Comment (JubayerJoy) [inline index.html:203]:
```suggestion
            <p class="adopter-description">and share your service mesh story with the community</p>
```

File: index.html [hunk 6]
```diff
@@ -196,6 +196,18 @@ <h1 class="used-by">Used By</h1>
...
+			<p class="button adopter-button">
+				<a href="https://github.com/layer5io/meshery/blob/master/ADOPTERS.md" target="_blank">
+					Become a Adopter
+				</a>
+			</p>
```
Comment (JubayerJoy) [inline index.html:208]:
```suggestion
			<div class="button adopter-button">
				<a href="https://github.com/layer5io/meshery/blob/master/ADOPTERS.md" target="_blank">
					Become a Adopter
				</a>
			</div>
```

Also, fix Indentation.

**Review by JubayerJoy — CHANGES_REQUESTED** [review]
Really nice work @jlcastrogro 👍
I would suggest some fixes to adjust this section with the rest of the page.
These changes will display as below.

screenshot-deploy-preview-369--mesheryio netlify app-2020 11 03-00_44_01

There is still some room for improvement, good luck with this 👍

File: index.html (hunk 6 above)
Comment (JubayerJoy) [reply index.html:208]:
```suggestion
			<p class="button adopter-button">
				<a href="https://github.com/layer5io/meshery/blob/master/ADOPTERS.md" target="_blank">
					Become an Adopter
				</a>
			</p>
```

Comment (JubayerJoy) [reply index.html:208]: P.S. Currently the section looks like this.

screenshot-deploy-preview-369--mesheryio netlify app-2020 11 03-00_55_00

File: _sass/landing-page.scss (hunk 1 above)
Comment (leecalcote) [reply _sass/landing-page.scss:120]: This suggestion has been implemented, correct, @jlcastrogro?

Comment (jlcastrogro) [reply _sass/landing-page.scss:120]: @leecalcote Do I take these suggestions?

Comment (jlcastrogro) [reply _sass/landing-page.scss:120]: @leecalcote I like more how looks it here, keeping gray color and letters uppercase
Screenshot_20201105_131101

**Review by leecalcote — APPROVED** [review]
Looking good, @jlcastrogro

**Review by JubayerJoy — APPROVED** [review]
@jlcastrogro Nice work :+1:

**Review by Neilblaze — APPROVED** [review]
Nice work! :100:

**Pull request merged**